doc/diagrams/c4-model/views.dsl

```diff
@@ -35,6 +35,13 @@ workspace extends "https://raw.githubusercontent.com/Energinet-DataHub/opengeh-a
             include ->certificatesDomain->
             autoLayout
         }
+        container energyOrigin "ContainerContextEO" {
+            title "[Container Context] Energy Origin"
+            include *
+            autoLayout
+        }
```

Comment:
What is the reason for having this view? I am not really sure what the point is of this PR...